### PR TITLE
#40 bug with interrupt update

### PR DESCRIPTION
**Pull request by DjKrys**
#40

### CODE REVIEW

File: src/sw/Core/src/GUI.cpp [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#include "tim.h"
+
```
Comment (LigasN) [inline src/sw/Core/src/GUI.cpp]: to much new lines

File: src/sw/Core/src/GUI.cpp [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
 #include "GUI.h"
```
Comment (LigasN) [inline src/sw/Core/src/GUI.cpp]: to much new lines

**Review by LigasN — CHANGES_REQUESTED** [review]

File: src/sw/Core/src/GUI.cpp (hunk 1 above)
Comment (DjKrys) [reply src/sw/Core/src/GUI.cpp]: fixed

File: src/sw/Core/src/GUI.cpp (hunk 2 above)
Comment (DjKrys) [reply src/sw/Core/src/GUI.cpp]: fixed

**Review by LigasN — APPROVED** [review]

**Pull request merged**